### ISSUE TITLE
[Raid] RS Buggs

### ISSUE THREAD

**Issue by elagen** [WoWManiaUK/Redemption#685]
there are some new bugs discovered on Halion

**1) Meteor Strike (75952) 

During Phase 1 of the fight The Meteor Strike should target a player and land then gives the flames on the floor - but once someone is targeted the
flames comes down first and then the meteor strikes on the floor - the flames should come after the meteor has landed.**

https://drive.google.com/open?id=0B_GODmkO59aEZzNfRG81ZmJEOTQ

As you can see in this video i have recorded 

Meteor Strike (75952)

2) Twilight cutter

   http://www.wowhead.com/spell=74769/twilight-cutter

During Phase 2 as we know we get twilight cutter which should kill people when they hit the cutter but as you can see in the video it does not kill
people but as you can also see in the video it does not kill people

3) Soul Consumption

In phase 2 in this video on 10 HC you can see the same damage as in 25 man making it harder and impossible to do

https://drive.google.com/open?id=0B_GODmkO59aES19Kb2JGT1dBVWM

Consumption - Deals 8750 to 11250 Shadow damage to targets within the shadow area == 25 Man damage - which i was receiving in 10 man
Consumption - Deals 2625 to 3375 Shadow damage to targets within the shadow area == 10 Man damage - which we should be receiving

http://wowwiki.wikia.com/wiki/Halion - 10 Man Halion
https://www.downfallguild.org/Halion - 25 Man Halion

4) Portal at the end of Phase 2 Bugged

Once phase 2 is done with there are portal at the start of phase 3 where half of the raid go back up to the physical realm and damage halion there
but in the video also you can see the portals back to the physical realm is bugged

https://drive.google.com/open?id=0B_GODmkO59aERFJzUlhSM01DQVE

as you can see in this video there are 2 people trying to go back through the portal and cannot enter the physical realm from the shadow realm and they cant.

**Comment by elagen** [comment]
And also on 25 HC - if the raid wipes on Halion he does not reset - like he now does in the other raids

**Comment by Emcitementz** [comment]
I know there's still a problem about twilight cutters, it's not concerning the one in the post here but make a new issue about it so i'll look at it.
